packages/trader/src/App/Constants/routes-config.ts

```diff
@@ -2,9 +2,15 @@ import React from 'react';
 import { RouteComponentProps } from 'react-router';
 import { routes, moduleLoader } from '@deriv/shared';
 import { localize } from '@deriv/translations';
-import Trade from 'Modules/Trading';
 import { TRouteConfig } from 'Types';
 
+const Trade = React.lazy(
```

Comment:
@farhan-nurzi-deriv this change doesn't seem to be necessary. the PR is for testing, not changing the way of loading the main trade page. :)  please check.
Such change would require a big sanity for DTrader.

Reply:
I will revert it back :)